routes/boxHistory.js

```diff
@@ -3,11 +3,15 @@ const express = require('express');
 const boxHistoryRouter = express();
 const {
   updateBox,
+  approveBoxInBoxHistory,
```

Comment:
🚫 **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'approveBoxInBoxHistory' is assigned a value but never used.